controllers/event/index.js

```diff
@@ -1,16 +1,22 @@
-import models from '../../models';
-import slugGenerator from '../../helpers/slugGenerator';
-import eventStatuschecker from '../../helpers/eventHelper/eventStatuschecker';
-import uploadCloudinary from '../../helpers/eventHelper/uploadCloudinary';
-import getEvents from '../../helpers/eventHelper/getEvent';
-import handleLikeUnlike from '../../helpers/eventHelper/handleLikeUnlike';
+import eventStatuschecker from "../../helpers/eventHelper/eventStatuschecker";
+import getEvents from "../../helpers/eventHelper/getEvent";
+import handleLikeUnlike from "../../helpers/eventHelper/handleLikeUnlike";
+import uploadCloudinary from "../../helpers/eventHelper/uploadCloudinary";
+import geocode from "../../helpers/googleMap/goecode";
+import slugGenerator from "../../helpers/slugGenerator";
+import models from "../../models";
+const { Op } = require("sequelize");
 const { Event, Likes, Ticket, TicketCategory } = models;
-
+var INCLUDETICKET = [
```

Comment:
Hey @Kasulejoseph , why var?

Reply:
`Scoping. ` 
**var** is a global scoped whereas **let** and **const** are blocked scope.
I used a functional programming paradigm in this file, and declaring the constant before all these functions with var is the best implement because of its scope. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Variable_scope